mcs/class/Makefile

```diff
@@ -424,6 +424,9 @@ wasm_tools_PARALLEL_SUBDIRS := $(wasm_tools_dirs_parallel)
 netcore_SUBDIRS := System.Private.CoreLib
 netcore_PARALLEL_SUBDIRS :=
 
+monodroid_designer_SUBDIRS := $(mobile_common_dirs)
+monodroid_designer_PARALLEL_SUBDIRS := $(monodroid_dirs_parallel)
```

Comment:
This could be "minimized", using our knowledge that not all assemblies are platform-independent and some can be re-used by the designer from the monodroid profile. But that will be brittle.